src/coord/src/catalog.rs

```diff
@@ -1666,6 +1668,22 @@ impl sql::catalog::Catalog for ConnCatalog<'_> {
         self.catalog.cluster_id
     }
 
+    fn search_path(&self, include_system_schemas: bool) -> Vec<&str> {
+        if include_system_schemas {
+            self.search_path.to_vec()
+        } else {
+            self.search_path
+                .iter()
+                .filter(|s| {
+                    (**s != PG_CATALOG_SCHEMA)
+                        & (**s != MZ_CATALOG_SCHEMA)
+                        & (**s != MZ_TEMP_SCHEMA)
```

Comment:
nit: I think you probably meant to use `&&` instead of `&` here? Don't think this actually impacts the semantics, but weird!